nextcloud_install_production.sh

```diff
@@ -704,9 +704,14 @@ rm -f *.zip.*
 done;
 
 # Install virtual kernels
-apt install linux-tools-virtual-hwe-16.04-edge linux-cloud-tools-virtual-hwe-16.04-edge  -y
-apt install linux-image-virtual-hwe-16.04-edge -y
-apt install linux-virtual-hwe-16.04-edge -y
+apt purge -y $(dpkg-query -W -f='${binary:Package}\n' 'linux-image-4*' 'linux-headers-4*' | grep -v 4.4.0-64-generic)
```

Comment:
Yes, it's the 4.4 kernel. Why can't we use the 4.8 kernel? It's a newer kernel and logically it should be even better...